website/docs/installation/upgrade.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Upgrade authentik
```

Comment:
This should really go into the "operating" category which we still haven't got

Reply:
Yeah, I could see it going in Operators section... but I think people tend to look for it alongside the Install docs...